apps/website/src/components/Wallet.vue

```diff
@@ -50,6 +67,17 @@ const {
   sendTransaction,
 } = useWallet()
```

Comment:
You could keep iopay in it's own composable file, but can you rework this so it's imported to/exported from the core wallet composable? Maybe we make `selectedProvider` a string (what you originally had) between MetaMask, CoinbaseWallet, IoPay.

Reply:
@shanejearley - Made some changes. Take a look and let me know what you think. In the meantime, I will move on to Phantom then Kepler (or vice versa depending on my research). 

My dad's actual birthday is tomorrow and I may be spending some day time with family. Ping me on slack, text, or call me if you want to talk. I'll be available to do those things at the very least.